src/Microsoft.Data.Sqlite/SqliteConnection.cs

```diff
@@ -170,25 +168,10 @@ public override void Open()
             MarshalEx.ThrowExceptionForRC(rc, _db);
 
             SetState(ConnectionState.Open);
-
-            OnOpened();
         }
 
-        partial void OnOpened();
-
 #if !NET451
-        partial void OnOpened()
-        {
-            var appDataType = CurrentApplicationData?.GetType();
-            var temporaryFolder = appDataType?.GetRuntimeProperty("TemporaryFolder").GetValue(CurrentApplicationData);
-            var temporaryFolderPath = temporaryFolder?.GetType().GetRuntimeProperty("Path").GetValue(temporaryFolder) as string;
-            if (temporaryFolderPath != null)
-            {
-                DbConnectionExtensions.ExecuteNonQuery(this, "PRAGMA temp_store_directory = '" + temporaryFolderPath + "';");
-            }
-        }
-
-        private static object CurrentApplicationData
+        internal static object CurrentApplicationData
```

Comment:
Instead of making this property internal, we should probably move it into its own helper/utility class.